README.md

```diff
@@ -51,9 +33,24 @@ roseus hrp2jsknt-interface.l
 
 ### Package Description
 
-hrpsys_ros_bridge_tutorialsls
+### [hrpsys_ros_bridge_tutorials]
 --------------
-This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common). 
+This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common).
 
 Euslisp tutorial wiki is written in [ROS wiki](http://wiki.ros.org/rtmros_common/Tutorials/WorkingWithEusLisp).
 
+### [hrpsys_gazebo_tutorials]
+
+This package is a collection of examples for using hrpsys_gazebo system and utility scripts.
+
+- You should prepare robot model file. Supported types of model file are collada(openrave) and VRML(openhrp3). URDF and OpenRAVE xml can be used by converting to collada.
+    - *&lt;robot_name&gt;*.yaml for configurating gazebo setting and hrpsys setting
+    - (automatically generated) *&lt;robot_name&gt;*.urdf under robot_models/*&lt;robot_name&gt;* directory
+    - (automatically generated) hrpsys settings (you should have a collada or VRML robot model file)
+    - *&lt;robot_name&gt;*_optional_urdf_setting.sh under robot_models/*&lt;robot_name&gt;* directory, this is for adding description used by gazebo (such as sensor settings, collision and friction setting)
+
+#### (automatically generated files)
+
+You can use robot_models/install_robot_common.sh for installing urdf model file. This scripts converts collada file in [hrpsys_ros_bridge_tutorials]/models directory to urdf file. 
```

Comment:
I couldn't find `robot_models/install_robot_common.sh`.

Reply:
Me too. (This is simply copied from rtmros_gazebo)
I removed this section.